### ISSUE TITLE
Bug in PIL when using vis.image in Python?

### ISSUE THREAD

**Issue by shelper** [facebookresearch/visdom#48]
when i run:
```
import visdom
import numpy as np
vis = visdom.Visdom()
vis.text('Hello, world!')
vis.image(np.ones((10, 10, 3)))
```

i got:
```
In [29]: import visdom
    ...: import numpy as np
    ...: vis = visdom.Visdom()
    ...: vis.text('Hello, world!')
    ...: vis.image(np.ones((10, 10, 3)))
    ...:
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
/Users/zyuan/anaconda/lib/python3.5/site-packages/PIL/Image.py in fromarray(obj, mode)
   2194             typekey = (1, 1) + shape[2:], arr['typestr']
-> 2195             mode, rawmode = _fromarray_typemap[typekey]
   2196         except KeyError:

KeyError: ((1, 1, 10), '|u1')

During handling of the above exception, another exception occurred:

TypeError                                 Traceback (most recent call last)
<ipython-input-29-94aa5cfce374> in <module>()
      3 vis = visdom.Visdom()
      4 vis.text('Hello, world!')
----> 5 vis.image(np.ones((10, 10, 3)))

/Users/zyuan/anaconda/lib/python3.5/site-packages/visdom/__init__.py in image(self, img, win, env, opts)
    307
    308         img = np.transpose(img, (1, 2, 0))
--> 309         im = Image.fromarray(img)
    310         buf = BytesIO()
    311         im.save(buf, format='JPEG', quality=opts['jpgquality'])

/Users/zyuan/anaconda/lib/python3.5/site-packages/PIL/Image.py in fromarray(obj, mode)
   2196         except KeyError:
   2197             # print(typekey)
-> 2198             raise TypeError("Cannot handle this data type")
   2199     else:
   2200         rawmode = mode

TypeError: Cannot handle this data type
```

**Comment by ajabri** [comment]
Oops, sorry this is because I changed the API to expect CxHxW instead of HxWxC. Forgot to change the example. It should be:

vis.image(np.ones((3, 10, 10)))

I can make this change later tonight.

**Comment by ajabri** [comment]
Fixed in 4e45bca
